Cookbook/Proposals/Base class for FeedbackLoops systems.md

```diff
@@ -0,0 +1,157 @@
+# Base class for FeedbackLoops systems
+
+* Author(s): Sergey Shulga
+* Review Manager: Anders Ha
+
+## Introduction
+
+Almost for all our screens, we implement ViewModels which are powered by the [ReactiveFeeedback](https://github.com/Babylonpartners/ReactiveFeedback).
+
+The current interface of VM looks like this
```

Comment:
so how will it change? Will `state.mutate` be the only change with this approach or the way we initialise state will also change? I suppose it will because the renderer signature changes now.

Reply:
Could be very likely that we won't have `Renderer` any more

Reply:
what we will have instead then?

Reply:
```
protocol View: Renderable where Self.View == Never {
    func body() -> AnyRenderable
}

Reply:
I don't think this is related to this proposal